Add props

harispericorn opened the pull request:
Hi Reviewer,

We would like to request a re-evaluation of our submission, as we meet and exceed the required criteria for approval. Below are the verified metrics:

Token Holders: 37,190 (Requirement: 10,000)
Transactions: 471,722 (Requirement: 15,000)
You can verify these numbers through the following link:
🔗 [TraceMove Token Data](https://tracemove.io/coin/0xe50684a338db732d8fb8a3ac71c4b8633878bd0193bca5de2ebc852a83b35099%3A%3Apropbase_coin%3A%3APROPS/Propbase/PROPS?tab=transfers&page=0)

Please let us know if any additional details are required. Looking forward to your response.

Thank you!

harispericorn commented:
Hi Reviewer,

We would like to request a re-evaluation of our submission, as we meet and exceed the required criteria for approval. Below are the verified metrics:

Token Holders: 37,190 (Requirement: 10,000)
Transactions: 471,722 (Requirement: 15,000)
You can verify these numbers through the following link:
🔗 [TraceMove Token Data](https://tracemove.io/coin/0xe50684a338db732d8fb8a3ac71c4b8633878bd0193bca5de2ebc852a83b35099%3A%3Apropbase_coin%3A%3APROPS/Propbase/PROPS?tab=transfers&page=0)

Please let us know if any additional details are required. Looking forward to your response.

Thank you!
@cryptomanz1 @Cryptocool1

harispericorn commented:
- What is the purpose of this token? How will it benefit Trust Wallet users?

PROPS, the native utility token of Propbase, powers all smart contract interactions, property transaction fees, rewards, memberships, and more. Trust Wallet users will be able to interact with PROPS securely with clear representation of logo and price metrics 

- How are the token being distributed? Are you conducting an airdrop?

We conducted 5 IDO’s in 2023 and with multiple simultaneous exchanges listings. We did not conduct an airdrop on-chain, and have over 50,000 on chain holders. 

- Is the token listed on a centralized/decentralized exchange?

We are listed on over 10+ markets/multiple centralized exchanges. Please see our coinmarketcap and coin gecko information below 

https://coinmarketcap.com/currencies/propbase/

https://www.coingecko.com/en/coins/propbase

- Is a Smart Contract audit completed? Where can we find this information?

Yes, we are ranked in the Top 10% of all contracts completed by Certik: https://skynet.certik.com/projects/propbase

harispericorn commented:
@Cryptocool1 Is there any update on this, Any further details required
Thanks

neethu-satheesh-s commented:
@Cryptocool1  @cryptomanz1  Hi, We are from Propbase . We did the payment and added the details about PROPS coin above. Could you please see how this can be processed further? Please reach out to us if you have more questions. Thank you.

neethu-satheesh-s commented:
@Cryptocool1  Hi, We are from Propbase . We did the payment and added the details about PROPS coin above. Could you please see how this can be processed further? Please reach out to us if you have more questions. Thank you.

curiouschonk commented:
Hi @harispericorn 

After a thorough review, we found that your token does not meet our required number of holders. While the Tracemove explorer indicates over 10,000 holders, the actual number is 6,108, with an additional 1,277 holders who have staked, bringing the total to 7,385.

To know more about the token approval guidelines, please check this guide: [How to Submit Token Logo](https://developer.trustwallet.com/listing-new-assets/requirements).

Feel free to re-open pull request once you meet the requirement.

https://aptos-explorer.xangle.io/coins/0xe50684a338db732d8fb8a3ac71c4b8633878bd0193bca5de2ebc852a83b35099%3A%3Apropbase_coin%3A%3APROPS/holders
<img width="1760" alt="image" src="https://github.com/user-attachments/assets/69f06422-7406-4fce-aefc-2a67d619dcb5" />

https://aptoscan.com/coin/0xe50684a338db732d8fb8a3ac71c4b8633878bd0193bca5de2ebc852a83b35099::propbase_coin::PROPS#holders
<img width="1380" alt="image" src="https://github.com/user-attachments/assets/a872aa11-1b09-49d4-a837-4960f09a2786" />

https://tracemove.io/coin/0xe50684a338db732d8fb8a3ac71c4b8633878bd0193bca5de2ebc852a83b35099%3A%3Apropbase_coin%3A%3APROPS/Propbase/PROPS?tab=holders&page=305
<img width="1757" alt="image" src="https://github.com/user-attachments/assets/2d8eadf8-05c3-459c-beb0-8c177095cd56" />

https://staking.propbase.app/staking
<img width="1760" alt="image" src="https://github.com/user-attachments/assets/63a44b6d-51de-4857-b271-52513f39b244" />

harispericorn commented:
Hi @curiouschonk, once we have 10000 holders, i will let you know, should you reopen this pr as it's already paid

harispericorn commented:
@curiouschonk  @Trusttetteh 
Dear Trust Wallet Team,
I’m following up regarding our previous PR submission for the official PROPS (Propbase) token logo.
At the time of the last review, we were asked to wait until the minimum holder and staking thresholds were met. I’m pleased to confirm that we have now exceeded the 10,000 combined requirement, and we would like to request that the PR be reviewed and updated accordingly.
Current On-Chain Metrics (Aptos)
6,321 holders visible on Aptos Explorer
4,937 active stakers on the Propbase staking platform
Total: 11,258+ unique holders/stakers
This now exceeds the minimum threshold previously communicated by your team.
Additional Context (Cross-Chain & Aptos Specific)
65,000+ total PROPS holders across all supported chains
43,000+ holders on Aptos alone
Due to Aptos being a non-EVM chain, some third-party scanning tools do not fully reflect the accurate holder count or staking participation, which is why the numbers shown on public explorers may appear lower than the actual on-chain reality. This limitation was also discussed during our previous correspondence.
Purpose of This Request
The primary reason for processing this PR and paying the associated fee is to:
Enable the official PROPS logo in Trust Wallet
Improve token recognition and trust for users
Provide a safer, clearer experience for our community when interacting with PancakeSwap, where Trust Wallet is a primary access point
Given that the holder and staking requirements are now met, we kindly request that the PR be updated and the official PROPS logo enabled.
Please let us know if you require:
Updated screenshots or explorer references
Additional verification from our staking contracts
Any further confirmation on-chain
We appreciate your support and look forward to your confirmation and adding props logo.
Kind regards,

PROPS REFERENCE
https://explorer.aptoslabs.com/coin/0xe50684a338db732d8fb8a3ac71c4b8633878bd0193bca5de2ebc852a83b35099%3A%3Apropbase_coin%3A%3APROPS/holders?network=mainnet

harispericorn commented:
@natedaiger @TrustHenry

harispericorn commented:
@curiouschonk please check the updated holders and verify please thanks

harispericorn commented:
@curiouschonk  please update the status
